fbpic/particles/deposition/cuda_methods.py

```diff
@@ -495,10 +495,10 @@ def deposit_J_gpu_linear(x, y, z, w, q,
 
 
 @cuda.jit
-def deposit_chi_gpu_linear(x, y, z, w, q, M, inv_gamma,
+def deposit_chi_gpu_linear_one_mode(x, y, z, w, q2_over_m_e0, inv_gamma,
                            invdz, zmin, Nz,
                            invdr, rmin, Nr,
-                           chi_tuple, m_tuple,
```

Comment:
As we recently discovered, tuples of arrays are not supported for gpu kernels ; therefore, I now call one kernel per mode.